vertx-mutiny-clients/vertx-mutiny-db2-client/src/test/java/io/vertx/mutiny/db2/DB2ClientTest.java

```diff
@@ -47,7 +56,7 @@ public void testMutinyAPI() {
                 .setUser(container.getUsername())
                 .setPassword(container.getPassword());
 
-        Pool client = DB2Pool.pool(vertx, options, new PoolOptions().setMaxSize(5));
+        Pool client = Pool.pool(vertx, options, new PoolOptions().setMaxSize(5));
```

Comment:
Shouldn't we use `DB2Pool`? (same question for all call sites related to pools, not matter the driver)